### ISSUE TITLE
Black Screenshots on Android in Release Mode

### ISSUE THREAD

**Issue by Taylor9876** [Instabug/Instabug-Flutter#232]
### Steps to Reproduce the Problem
We have updated our Flutter from 2.8.1 to 2.10.5 and in the new version of flutter the Android screenshots are no longer working. We are getting a black screen instead of the actual screen. This issue only occurs on release versions, local dev will create the expected screenshots.
### Expected Behavior
Screenshot of the content displayed.
### Actual Behavior
Black screenshot
### SDK Version
instabug_flutter: 10.13.0

### Flutter, iOS and Android Versions
Flutter 2.10.5
iOS any version
Android tested 7-12

**Comment by DavidMina96** [comment]
@Taylor9876 Thank you for reporting this. We're looking into it and will get back to you.

**Comment by DavidMina96** [comment]
@Taylor9876 We're working on a fix for this issue to be included in our next release. In the meantime, you can do the following as a temporary workaround to get unblocked. 

1. Add the below snippet to the app `build.gradle` file:

    ```gradle
    android {
    
        ...
    
        buildTypes {
    
            release {
    
                signingConfig signingConfigs.debug
                
                // Temporary fix for Instabug/Instabug-Flutter#232
                useProguard true
                proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
            }
        }
    } 
    ```
1. Create `/android/app/proguard-rules.pro` file and insert inside:
    ```
    -keep class com.instabug.bug.BugReporting {*;}
    ```

**Comment by Taylor9876** [comment]
@DavidMina96 Thank you for the quick reply. We will implement this fix pending the next release.

**Comment by BrahimAkar** [comment]
i have the same issue, any updates?

**Comment by DavidMina96** [comment]
@BrahimAkar Apologies for any inconvenience this may have caused. The fix is planned to be out in our upcoming release, and until then, the aforementioned workaround should resolve the issue for you. Please let me know if you have any feedback on it.

**Comment by DavidMina96** [comment]
Fixed in v11.0.0 🚀
